### ISSUE TITLE
Finetuned model weights

### ISSUE THREAD

**Issue by rohitrango** [THUDM/ImageReward#90]
Hi, 

Thank you for releasing code for the work! Have you released the weights for any finetuned model in the paper? 

Thanks.

**Comment by xujz18** [comment]
Thanks for your attention! You can construct your own prompt set to get a model that better meets your needs.
> #84

**Comment by rohitrango** [comment]
Why does the script `scripts/train_refl_sdxl.sh` require images too (`train_refl.sh` doesnt)?

**Comment by Cominclip** [comment]
> Why does the script `scripts/train_refl_sdxl.sh` require images too (`train_refl.sh` doesnt)?

I meet the same question, do you have solved it? i don't understand what the images refer to..

**Comment by xujz18** [comment]
This is code contributed by a community friend who supposedly added the pre-training loss function mentioned in the paper. You can refer to https://github.com/THUDM/ImageReward/pull/89
